gcp-cs-network-component/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Pulumi.Gcp.Compute;
+using Pulumi.Gcp.Compute.Inputs;
+
+class Program
+{
+    static Task<int> Main()
+    {
+        return Deployment.RunAsync(() =>
+        {
+            string instanceName = "test";
+            var network = new Pulumi.Gcp.Compute.Network(instanceName);
```

Comment:
```suggestion
            var network = new Network(instanceName);
```

We try not to use the full namespaces, I think you have all the required `using`'s above.

Please adjust everywhere.